.drone.yml

```diff
@@ -40,7 +40,7 @@ steps:
   depends_on: [ clone ]
   commands:
   - cp -R . /tmp/2/ && cd /tmp/2/
-  - ./project/scripts/sbt ";dotty-bootstrapped/compile ;dotty-bootstrapped/test ;dotty-staging/test ;sjsSandbox/run;sjsSandbox/test;sjsJUnitTests/test ;configureIDE"
+  - ./project/scripts/sbt ";dotty-bootstrapped/compile ;dotty-bootstrapped/test ;dotty-staging/test ;dotty-tasty-consumer/test ;dotty-tastydoc/test ;sjsSandbox/run;sjsSandbox/test;sjsJUnitTests/test ;configureIDE"
```

Comment:
If a project is aggregated in dotty-bootstrapped, then `dotty-bootstrapped/test` should run its tests, so no need to run them explicitly.

Reply:
Done. I also added `tastydoc` to the aggregation.